js/main.js

```diff
@@ -20,8 +51,49 @@ function logSubmit(event) {
     entryId: data.nextEntryId
   };
   data.nextEntryId++;
-  data.entries.unshift(entry);
+  entries.unshift(entry);
 
```

Comment:
one thing to add is that inside of this function, you need to render the new entry and use the prepend method to add it to the DOM, so when the view switches to the entries view, you can see the newest entry.

Reply:
I have both of these things happening in the event listener when DOM content is loaded (main.js lines 93-99). Do I have to delete that and place it inside this function instead?